kview/src/main/java/dev/ikm/komet/kview/lidr/mvvm/model/DataModelHelper.java

```diff
@@ -92,13 +96,36 @@ public class DataModelHelper {
             Entity.getFast(TinkarTerm.DESCRIPTION_INITIAL_CHARACTER_CASE_SENSITIVE.nid())
     );
 
+    public static final Set<ConceptEntity> DATA_TYPE_OPTIONS = Set.of(
+            Entity.getFast(TinkarTerm.STRING.nid()),
+            Entity.getFast(TinkarTerm.COMPONENT_FIELD.nid()),
+            Entity.getFast(TinkarTerm.COMPONENT_ID_SET_FIELD.nid()),
+            Entity.getFast(TinkarTerm.COMPONENT_ID_LIST_FIELD.nid()),
+            Entity.getFast(TinkarTerm.DITREE_FIELD.nid()),
+            Entity.getFast(TinkarTerm.DIGRAPH_FIELD.nid()),
+            Entity.getFast(TinkarTerm.CONCEPT_FIELD.nid()),
+            Entity.getFast(TinkarTerm.SEMANTIC_FIELD_TYPE.nid()),
+            Entity.getFast(TinkarTerm.INTEGER_FIELD.nid()),
+            Entity.getFast(TinkarTerm.FLOAT_FIELD.nid()),
+            Entity.getFast(TinkarTerm.BOOLEAN_FIELD.nid()),
+            Entity.getFast(TinkarTerm.BYTE_ARRAY_FIELD.nid()),
+            Entity.getFast(TinkarTerm.ARRAY_FIELD.nid()),
+            Entity.getFast(TinkarTerm.INSTANT_LITERAL.nid()),
+            Entity.getFast(TinkarTerm.LONG.nid())
+    );
```

Comment:
Can this just be the TinkarTerm items? 
There is no need to query every each entity.

Reply:
This will be require to cast EntityProxy.Concept to be casted to ConceptEntity. We should move away from EnitytProxy right ?

Reply:
All we really want is a file `DataModelHelper.java` in the package: `dev/ikm/komet/kview/mvvm/model` 
```java
// do a static import of TinkarTerm.XXXXX where xxxxx is each data type


public static final Set<EnitityFacade> DATA_TYPE_OPTIONS = Collections.unmodifiableSet(Set.of(
            STRING,
            COMPONENT_FIELD,
            COMPONENT_ID_SET_FIELD,
            COMPONENT_ID_LIST_FIELD,
            DITREE_FIELD,
            DIGRAPH_FIELD,
            CONCEPT_FIELD,
            SEMANTIC_FIELD_TYPE,
            INTEGER_FIELD,
            FLOAT_FIELD,
            BOOLEAN_FIELD,
            BYTE_ARRAY_FIELD,
            ARRAY_FIELD,
            INSTANT_LITERAL,
            LONG
    ));    
```

Reply:
Done.

Reply:
Thank you.